docs/state.md

```diff
@@ -7,8 +7,7 @@ sidebar_label: Handler State
 As described in ["About Gesture Handlers"](about-handlers.md) section gesture handlers can be treated as ["state machines"](https://en.wikipedia.org/wiki/Finite-state_machine).
 Each handler instance at any given time has an assigned state that can change when new touch events arrive or can be forced by the touch system to change it state in certain circumstances.
 
-There are six possible states for the handler:
- - [UNDETERMINED](#undetermined)
```

Comment:
ok, but if we remove it what is going to be the "oldState" when we go into "BEGAN"?

Reply:
I didn't wish to inform about this change.